### ISSUE TITLE
desperately need to be able to check-in attendees from a mobile phone

### ISSUE THREAD

**Issue by richardteachout** [eventespresso/eea-rest-api#42]
desperately need to be able to check-in attendees from a mobile phone, as we had in EE3.. 
So, basically, we're waiting on the EE4write endpoints.  50% of the entire reason we for eventespresso was for exactly that purpose (for our haunted house each sept/oct - but last oct we didn't have the ability to check-in attendees remotely easily (as the admin interface / bar code scanning doesn't work well from an iphone)... so it was a royal pain.  I would write my own interface into the system through the API if necessary (but a new APP would be amazing).. but without the write ability, it's seriously inhibiting us.

PLEASE MAKE THIS AVAILABLE by APRIL/MAY, so I do not need to look for an alternate software for the season this year.

Thanks!

**Comment by garthkoyle** [comment]
Hi Richard,

Thank you for your feedback. 

Are handheld bar/qr scanners not an option?

We have started development on mobile apps but I can't give you a firm completion date. So you want the mobile apps by May to use in September?

**Comment by mnelson4** [comment]
also @richardteachout have you seen we added a checkin/checkout endpoint in 4.8.33? Here's the documentation: http://developer.eventespresso.com/docs/ee4-rest-api-rpc-style-endpoints/. Does that meet your need?

**Comment by devlift** [comment]
Hi there,

Just wondering if there are any plans to add event creation and registration via the API. We are building a pickup sports app and your plugin fits perfectly, except for these two items.

Thanks!

**Comment by mnelson4** [comment]
we actually got started on this in EE core branch https://github.com/eventespresso/event-espresso-core/tree/FET-9222-rest-api-writes but put it on hold because we weren't aware of much interest. Would you be willing to help test it if we got that branch working again?

**Comment by nerrad** [comment]
> Hi there,
> Just wondering if there are any plans to add event creation and registration via the API. We are building a pickup sports app and your plugin fits perfectly, except for these two items.

This should go into its own github issue..

**Comment by devlift** [comment]
Thanks for the quick reply. We would absolutely be willing to test. @nerrad sorry - I can also create a separate issue.

Thanks!

**Comment by mnelson4** [comment]
yeah; and seeing how this feature will be in EE core, not the deprecated rest api addon, it would probably be best created in the EE core issue tracker.

And ya the original issue this is for has been addressed because you can currently check registrations in and out of events via the api

**Comment by devlift** [comment]
Thanks @mnelson4 - Is there a specific area where you would like be to add the feature request?

Thanks!

**Comment by mnelson4** [comment]
I'd suggest adding it here: https://github.com/eventespresso/event-espresso-core/issues
